### DISCUSSION TITLE
dot队现在强度如何（2卡1黑）

### DISCUSSION THREAD

**Discussion by peterquill163** [share121/inter-knot#4635] [崩坏：星穹铁道]
想问一下强度如何 就在混沌表现 顺便帮我看看练度咋样 感兴趣！
![0B08C403586DAE4202792F54EFD63ACB](https://github.com/user-attachments/assets/db4cd8da-e9d0-45dd-9a45-e2f28ba63ee2)

**Comment by canyusuo** [comment]
我也是2+1卡，现在dot队不太行了，三体人队伍随随便便10w+，传统卡鹅阮霍 卡芙卡吃满buff可能才能打个十几二十万真的比不过<img src="https://img-static.mihoyo.com/communityweb/upload/70b5b0c9543ee5080c6ae92fd2797892.png" alt="心情复杂" width="75" >
